lib/is.js

```diff
@@ -75,14 +75,24 @@ var util = require('./util');
  * Steam tests.
  */
 (function () {
-    var identifier = '[0-9\\-_]{1,100}';
+    var identifier = '[\\w-]{2,32}';
+    var numericIdentifier = '[0-9]{16}';
```

Comment:
Nevermind, that's not called an account name; it's called a Custom URL. https://steamcommunity.com/discussions/forum/1/537402115094224389/
https://steamcommunity.com/discussions/forum/1/496880503074638068/